common/changes/@uifabric/example-app-base/hotfix-AddFocus2Nav-office-ui-fabric-react-5.0_2020-01-22-04-47.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Please delete all the change files except for the one with office-ui-fabric-react in the name.

Reply:
Thanks, resolved.